### PR TITLE
cfengine 3.17.0

### PR DESCRIPTION
**Pull request by mprince**
Created with `brew bump-formula-pr`.

~~`resource` blocks may require updates.~~

### CODE REVIEW

File: Formula/cfengine.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 class Cfengine < Formula
   desc "Help manage and understand IT infrastructure"
   homepage "https://cfengine.com/"
-  url "https://cfengine-package-repos.s3.amazonaws.com/tarballs/cfengine-3.16.0.tar.gz"
-  sha256 "f4256e6e1ca04776a9fd48f1388a30edfa8d11fdcf870ba62ce5b0ad62a87372"
+  url "https://cfengine-package-repos.s3.amazonaws.com/tarballs/cfengine-3.17.0.tar.gz"
+  sha256 "b7def98c2bdf491839a6dfb85ac48c670136cf331984b6c4063551ed3daf1420"
+  license "MIT"
```
Comment (Moisan) [inline Formula/cfengine.rb:6]: The LICENSE file states
```
CFEngine is provided under the terms of the GNU General Public License version 3
(below), with explicit permission to link with the OpenSSL library, BerkeleyDB
library and and PCRE library.

On some systems, code under the Frontier Artistic License
(/libcompat/snprintf) might become compiled. This is compatible with the
GPL.

Users of the software may, at their option, choose the COSL license
below as part of the enterprise CFEngine product.
```

So this is clearly not MIT.

File: Formula/cfengine.rb [hunk 2]
```diff
@@ -1,8 +1,9 @@
 class Cfengine < Formula
   desc "Help manage and understand IT infrastructure"
   homepage "https://cfengine.com/"
-  url "https://cfengine-package-repos.s3.amazonaws.com/tarballs/cfengine-3.16.0.tar.gz"
-  sha256 "f4256e6e1ca04776a9fd48f1388a30edfa8d11fdcf870ba62ce5b0ad62a87372"
+  url "https://cfengine-package-repos.s3.amazonaws.com/tarballs/cfengine-3.17.0.tar.gz"
+  sha256 "b7def98c2bdf491839a6dfb85ac48c670136cf331984b6c4063551ed3daf1420"
+  license "GPL-3.0-only"
```
Comment (SMillerDev) [inline Formula/cfengine.rb:6]: It's also not GPL-3.0-only since it has exceptions.

**Comment by mprince** [comment]
@SMillerDev @Moisan I have removed the license block since it wasn't there and I am not sure how to deal with exceptions on the license block.

**Review by Moisan — APPROVED** [review]
Thank you @mprince ! We'll figure out the license another time.

**Pull request closed without merging**